nautilus_trader/data/aggregation.pxd

```diff
@@ -51,7 +51,7 @@ cdef class BarBuilder:
     cdef Price _close
     cdef Quantity volume
 
-    cpdef void set_partial(self, Bar partial_bar)
+    cpdef void set_partial(self, Bar partial_bar, bint run_once=*)
```

Comment:
Whats the functionality of this `run_once` param?

Reply:
it's to be able to run set_partial several times when it's equal to False